baseline/OSPS-BR.yaml

```diff
@@ -278,6 +296,17 @@ controls:
         identifiers:
           - 3.1
           - 3.5
+      - reference-id: 800-161
+        identifiers: 
+          - AU-2
+          - AU-6
+          - AU-10
+          - CM-5
+          - CM-6
```

Comment:
CM-6 seems to be more about the changes to the project's infrastructure, not the changes in the software itself. I could see an argument that this Baseline control implements CM-6 for downstream consumers, though, so I don't know if we should keep this or not.